tests/cases/fourslash/indentationInJsx3.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path='fourslash.ts' />
+
+//@Filename: file.tsx
+////function foo () {
+////   return (
+////        <div>
+////hello
+////goodbye
+////        </div>
+////    )
+////}
+
+goTo.position(21);

```

Comment:
use a marker instead

Reply:
Or just use `verify.currentFileContentIs` instead of `textAtCaretIs`.